themes/src/main/resources-community/theme/base/account/messages/messages_fi.properties

```diff
@@ -138,7 +138,7 @@ revoke=Kumoa oikeudet
 
 configureAuthenticators=Konfiguroitu kaksivaiheinen kirjautuminen
 mobile=Mobiili
-totpStep1=Asenna <a href="https://freeotp.github.io/" target="_blank">FreeOTP</a> tai Google Authenticator ohjelma laiteellesi. Kummatkin sovellukset ovat saatavilla <a href="https://play.google.com">Google Play</a> ja Apple App Store kaupoissa.
+totpStep1=Asenna jokin seuraavista sovelluksista matkapuhelimeesi:
```

Comment:
This change (the same for other locales) is keeping it consistent with the same message in `_en` ?

Reply:
Yes ... the message in `_en.properties` has been changed. As the validator now ensure that no translation adds additional llinks for security reasons, these translations need to change in this PR.